bids-validator/utils/bids_files.js

```diff
@@ -56,24 +56,31 @@ function checkFileListForMatch(i) {
  */
 function verifyDatafileMatch(sidecarPath, noExt, matchFile) {
   let match = false
-  let megDs = false
+  let folderMatch = false
+
   // Make sure it's not the data dictionary itself
   const isSelf = matchFile.relativePath === sidecarPath
   if (!isSelf && type.file.isDatafile(matchFile.relativePath)) {
     match = true
   }
 
-  // MEG datafiles may be a folder, therefore not contained in fileList, will need to look in paths
+  // IEEG and MEG datafiles may be a folder, therefore not contained in fileList, will need to look in paths
   if (
     !isSelf &&
-    !match &&
-    (noExt.endsWith('_meg') || noExt.endsWith('_coordsystem'))
-  ) {
-    megDs = matchFile.relativePath.includes('_meg.ds')
+    !match) {
+
+    if (noExt.endsWith('_meg') || noExt.endsWith('_coordsystem')) {
+      folderMatch = matchFile.relativePath.includes('_meg.ds')
+    }
+    if (noExt.endsWith('_ieeg') || noExt.endsWith('_coordsystem')) {
+      folderMatch = matchFile.relativePath.includes('_ieeg.mefd')
+    }
+	
```

Comment:
I think ESLint is [complaining](https://app.circleci.com/pipelines/github/bids-standard/bids-validator/92/workflows/302a4a6f-c322-4405-9175-8904960564bf/jobs/4961/steps) about these extra newlines:

```suggestion
    if (noExt.endsWith('_meg') || noExt.endsWith('_coordsystem')) {
      folderMatch = matchFile.relativePath.includes('_meg.ds')
    }
    if (noExt.endsWith('_ieeg') || noExt.endsWith('_coordsystem')) {
      folderMatch = matchFile.relativePath.includes('_ieeg.mefd')
    }
```